test/PartnerProxyFactory.spec.ts

```diff
@@ -41,58 +36,54 @@ async function initialSetup() {
   const partner2 = signers[2];
   const nameOwner = signers[3];
 
-  const RNS = await deployMockContract<$RNS>(owner, RNSJson.abi);
+  const RNS = await deployMockContract<RNSType>(RNSJson.abi);
 
-  const Resolver = await deployMockContract<$Resolver>(owner, ResolverJson.abi);
+  const Resolver = await deployMockContract<ResolverType>(ResolverJson.abi);
 
-  const NodeOwner = await deployMockContract<$NodeOwner>(
-    owner,
-    NodeOwnerJson.abi
-  );
+  const NodeOwner = await deployMockContract<NodeOwnerType>(NodeOwnerJson.abi);
 
-  const RIF = await deployMockContract<$RIF>(owner, RIFJson.abi);
+  const RIF = await deployMockContract<RIFType>(RIFJson.abi);
 
-  const PartnerConfiguration = await deployMockContract<$IPartnerConfiguration>(
-    owner,
-    IPartnerConfigurationJson.abi
-  );
+  const PartnerConfiguration =
+    await deployMockContract<IPartnerConfigurationType>(
+      IPartnerConfigurationJson.abi
+    );
 
-  const FeeManager = await deployMockContract<IFeeManager>(
-    owner,
+  const FeeManager = await deployMockContract<IFeeManagerType>(
     IFeeManagerJson.abi
   );
 
-  const PartnerManager = await deployMockContract<$PartnerManager>(
-    owner,
+  const PartnerManager = await deployMockContract<PartnerManagerType>(
     PartnerMangerJson.abi
   );
 
-  const { contract: PartnerRegistrar } =
-    await deployContract<$PartnerRegistrar>('$PartnerRegistrar', {
-      NodeOwner: NodeOwner.address,
-      RIF: RIF.address,
-      IPartnerManager: PartnerManager.address,
-      RNS: RNS.address,
-      rootNode: tldNode,
-    });
-
-  const { contract: PartnerProxy } = await deployContract<$PartnerProxy>(
-    '$PartnerProxy',
-    {}
+  const { contract: PartnerRegistrar } = await deployContract(
+    'PartnerRegistrar',
+    {
+      _nodeOwner: NodeOwner.address,
```

Comment:
There's inconsistency with these field names with the others

Reply:
Missed this one, thanks for catching it